### ISSUE TITLE
coordinateCoverter is missing in types

### ISSUE THREAD

**Issue by perotom** [gagan-bansal/geojson2svg#36]
Type checker throws errors

**Comment by gagan-bansal** [comment]
@perotom Thanks for reporting I'll check this.

**Comment by nickcoutsos** [comment]
Looks like this was fixed in #37, will you be publishing a new release of the package on NPM?

**Comment by gagan-bansal** [comment]
I had published on npm also, I'll check the issue again today.

**Comment by gagan-bansal** [comment]
Yes, I missed it, I have published now. Please check and let me know if there is any issue.
